com.github.OpenHV.yaml

```diff
@@ -0,0 +1,63 @@
+app-id: com.github.OpenHV
```

Comment:
this is not a proper app-id for a github hosted. It should have 4 components, where number 3 is the org/user and number 4 is the app.